### ISSUE TITLE
rebase main signing branch on top of latest dev

### ISSUE THREAD

**Issue by zkat** [NuGet/Home#9012]
It's been a while since we've done a rebase, so once NuGet/NuGet.Client#3168 is merged into `dev-feature-xplatVerification`, we should go do that.

**Comment by zkat** [comment]
https://github.com/NuGet/NuGet.Client/pull/3191 has been merged, so this is done!
